src/coffee/stockimport.coffee

```diff
@@ -371,8 +371,18 @@ class StockImport
             .fetch()
             .then (result) =>
               @_updateInventory(entry, result.body, tryCount + 1)
+            .catch (err) =>
+              if (err.statusCode == 404)
+                debug "Stock not existing anymore (probably was deleted), create a new stock"
+                @client.inventoryEntries.create(entry)
+              else
+                debug "Error on refetching stock on 409 error. Details: #{JSON.stringify(err)}"
+                Promise.reject err
           else
-            Promise.reject new Error("Retry limit #{max409Retries} reached for stock #{JSON.stringify(entry)}")
+            Promise.reject new Error("Retry limit #{@max409Retries} reached for stock #{JSON.stringify(entry)}")
```

Comment:
Lets be consistent with what we have for sphere-node-utils already:
`Failed to retry handling 409 error after #{@max409Retries} attempts for stock #{JSON.stringify(entry)}`

Reply:
https://github.com/sphereio/sphere-stock-import/pull/69/commits/f0227ce01505b32c5565cc6d784389281b01533d